### PR TITLE
[oap-cache/oap]Modified test case configs of oap-perf-suite

### PR DESCRIPTION
**Pull request by HongW2019**
## What changes were proposed in this pull request?

Add test case configs to oap-perf-suite.

## How was this patch tested?
Passed the daily test.

### CODE REVIEW

File: src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala [hunk 1]
```diff
@@ -382,27 +773,27 @@ trait LocalClusterConfigSet extends BenchmarkConfigSelector {
...
-//      .setSparkConf("spark.yarn.executor.memoryOverhead", "1g")
-//      .setSparkConf("spark.executor.memory", "110g")
-//      .setSparkConf("spark.sql.oap.onheap.size", "10g")
-//      .setSparkConf("spark.sql.oap.offheap.enable", "false")
+    // is a object.
+    //    new BenchmarkConfig()
+    //      .setBenchmarkConfName("executor on/off heap: 100/0")
+    //      .setBenchmarkConf(BenchmarkConfig.FILE_FORMAT, "oap")
+    //      .setBenchmarkConf(BenchmarkConfig.INDEX_ENABLE, "true")
+    //      .setSparkConf("spark.memory.offHeap.enabled", "true")
```
Comment (zhixingheyi-tian) [inline src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala:783]: Just space diffs?

File: src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala [hunk 2]
```diff
@@ -132,7 +132,7 @@ object BenchmarkConfigSelector {
     if (wildcardConfiguration.nonEmpty) {
       wildcardConfiguration.exists{conf =>
         config.getConf(conf._1) == conf._2 ||
-        config.confName.equals(conf._2)
+          config.confName.equals(conf._2)
       }
```
Comment (zhixingheyi-tian) [inline src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala:136]: Space？

File: src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala (hunk 1 above)
Comment (HongW2019) [reply src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala]: Modified

File: src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala (hunk 2 above)
Comment (HongW2019) [reply src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala]: Modified.

File: src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala [hunk 3]
```diff
@@ -171,51 +171,168 @@ trait ParquetVsOapVsOrcConfigSet extends BenchmarkConfigSelector{
...
+      .setBenchmarkConfName("Orc w/ index pm")
+      .setBenchmarkConf(BenchmarkConfig.FILE_FORMAT, "orc")
+      .setBenchmarkConf(BenchmarkConfig.INDEX_ENABLE, "true")
+      .setSparkConf("spark.sql.oap.fiberCache.memory.manager", "pm")
+      .setSparkConf("spark.oap.cache.strategy", "guava"),
+    new BenchmarkConfig()
+      .setBenchmarkConfName("Orc w/ index vmem")
+      .setBenchmarkConf(BenchmarkConfig.FILE_FORMAT, "orc")
+      .setBenchmarkConf(BenchmarkConfig.INDEX_ENABLE, "true")
+      .setSparkConf("spark.sql.oap.fiberCache.memory.manager", "tmp")
```
Comment (winningsix) [inline src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala:188]: @HongW2019 This PR will reduce the configuration for memory manager settings. See https://github.com/Intel-bigdata/OAP/pull/1179 You can place a TODO here to simplify your settings and update it once we got it merged. cc @yma11

Comment (HongW2019) [reply src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala]: @winningsix Have removed memory manager settings from oap-perf-suite, so changing these configs in spark-defaults.conf can be more convenient according to different daily test needings.

Comment (winningsix) [reply src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala]: LGTM Thanks!

**Comment by gczsjdy** [comment]
@HongW2019 You may want to create a new PR from a new branch to trigger travis. : )

**Comment by winningsix** [comment]
@gczsjdy can we still use this PR to trigger?

**Comment by gczsjdy** [comment]
@winningsix I'm afraid not... : )

**Comment by winningsix** [comment]
@gczsjdy it works now :) you can just push a commit.

**Comment by winningsix** [comment]
@HongW2019 Thanks for update. it looks good to me for the new changes.

File: src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala [hunk 4]
```diff
@@ -184,22 +184,39 @@ trait ParquetVsOapVsOrcConfigSet extends BenchmarkConfigSelector{
...
+      .setSparkConf("spark.sql.oap.fiberCache.memory.manager", "mix")
+      .setSparkConf("spark.oap.cache.strategy", "mix"),
+    new BenchmarkConfig()
+      .setBenchmarkConfName("Orc w/ index data binary cache separation different medium enabled")
+      .setBenchmarkConf(BenchmarkConfig.FILE_FORMAT, "orc")
+      .setBenchmarkConf(BenchmarkConfig.INDEX_ENABLE, "true")
+      .setSparkConf("spark.sql.oap.orc.binary.cache.enable", "true")
+      .setSparkConf("spark.sql.oap.index.data.cache.separation.enable", "true")
+      .setSparkConf("spark.sql.oap.fiberCache.memory.manager", "mix")
+      .setSparkConf("spark.oap.cache.strategy", "mix"),
```
Comment (zhixingheyi-tian) [inline src/test/oap-perf-suite/scala/org/apache/spark/sql/BenchmarkConfig.scala:219]: Consider how to avoid testing these cases in DRAM scenario

**Pull request merged**